internal/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -478,8 +493,20 @@ func (c *enrollCmd) enrollWithBackoff(ctx context.Context, persistentConfig map[
 
 	c.log.Infof("Starting enrollment to URL: %s", c.client.URI())
 	err := c.enroll(ctx, persistentConfig)
+	if err == nil {
+		return nil
+	}
+
+	const deadline = 10 * time.Minute
+	const frequency = 60 * time.Second
```

Comment:
This feels too slow for me. We can retry faster than once every minute.

I also think waiting for 10 minutes is too long, we should fail earlier than this.

Right now the enroll command is going to appear like it is hung since there won't be any user feedback faster than every 1 minute.

Reply:
Ah I see this is what it was before for the Fleet URL, I care more about the new backoff  for the restart command on the same host. Commented in the wrong place.